sdk/cosmos/tests.yml

```diff
@@ -22,7 +22,6 @@ jobs:
           DisplayName: ReadMyWrites Integration Tests
           OSVmImage: 'ubuntu-18.04'
           ProfileFlag: '-Pe2e'
-          AdditionalArgs: '-DargLine="-Dcosmos.directModeProtocol=Https"'
```

Comment:
why these ones are removed? is that intentional?

Reply:
Yes, currently on V3-  we are not running e2e tests on Tcp mode, which is important as compared to running e2e tests on https mode.